tests/runtest.cmd

```diff
@@ -132,6 +136,68 @@ set "__TestWorkingDir=%__RootBinDir%\tests\%__BuildOS%.%__BuildArch%.%__BuildTyp
 if not defined XunitTestBinBase       set  XunitTestBinBase=%__TestWorkingDir%
 if not defined XunitTestReportDirBase set  XunitTestReportDirBase=%XunitTestBinBase%\Reports\
 
+REM At this point in the script there will be a divergence in how the tests are run
+REM For official builds we will continue to run tests using the un-unified scripting
+REM which relies on msbuild and calls runtest.proj directly. For all other scenarios
+REM runtest.py will handle setup and calls runtest.proj.
+
+if defined __AgainstPackages goto SetupMSBuildAndCallRuntestProj
+if "%__CoreFXTests%"=="true" goto SetupMSBuildAndCallRuntestProj
+if "%__PerfTests%"=="true" goto SetupMSBuildAndCallRuntestProj
+if defined __GenerateLayoutOnly goto SetupMSBuildAndCallRuntestProj
+
+REM We are not running in the official build scenario, call runtest.py
+
+set __RuntestPyArgs=-arch %__BuildArch% -build_type %__BuildType%
+
+if defined DoLink (
+  set __RuntestPyArgs=%__RuntestPyArgs% --il_link
+)
+
+if defined __LongGCTests (
+    set __RuntestPyArgs=%__RuntestPyArgs% --long_gc
+)
+
+if defined __GCSimulatorTests (
+    set __RuntestPyArgs=%__RuntestPyArgs% --gcsimulator
+)
+
+if defined __JitDisasm (
+    set __RuntestPyArgs=%__RuntestPyArgs% --jitdisasm
+)
+
+if defined __IlasmRoundTrip (
+    set __RuntestPyArgs=%__RuntestPyArgs% --ilasmroundtrip
+)
+
+if defined __TestEnv (
+    set __RuntestPyArgs=%__RuntestPyArgs% -test_env %__TestEnv%
```

Comment:
So most runtest.py arguments use 2 dashes, but `-test_env` only uses one?

Reply:
This follows the convention that one dash expects a following argument, while two dashes are a flag.